plugins/source_scheduler_dfs.go

```diff
@@ -102,14 +102,14 @@ func (p *SourcePlugin) resolveTableDfs(ctx context.Context, allIncludedTables sc
 				})
 				p.logger.Error().Interface("error", err).Str("stack", stack).Msg("table resolver finished with panic")
 				atomic.AddUint64(&tableMetrics.Panics, 1)
+				close(res) // assuming the panic is from the resolver, don't leave the channel open
```

Comment:
I guess the panic could have also been from closing the channel, in which case we'd be causing another panic here 😅